[C,application/controllers/admin/surveyaction.php,134]         $clang = $this->getController()->lang;
[C,application/controllers/admin/surveyaction.php,135] 
[A,application/controllers/admin/surveyaction.php,136]         $_SESSION['FileManagerContext'] = "edit:survey:{$surveyid}";
[A,application/controllers/admin/surveyaction.php,137] 
[C,application/controllers/admin/surveyaction.php,138]         $esrow = array();
[C,application/controllers/admin/surveyaction.php,139]         $esrow = self::_fetchSurveyInfo('editsurvey', $surveyid);
[D,application/controllers/admin/surveyaction.php,138]         //$esrow = $esrow[0]->tableSchema->columns;
[C,application/controllers/admin/surveyaction.php,140]         $aData['esrow'] = $esrow;
[C,application/controllers/admin/surveyaction.php,141] 
[C,application/controllers/admin/surveyaction.php,142]         $aData = array_merge($aData, $this->_generalTabEditSurvey($surveyid, $esrow));
@@ -146,8 +147,6 @@ function editsurveysettings($surveyid)
[C,application/controllers/admin/surveyaction.php,147]         $aData = array_merge($aData, $this->_tabPanelIntegration($esrow));
[C,application/controllers/admin/surveyaction.php,148]         $aData = array_merge($aData, $this->_tabResourceManagement($surveyid));
[C,application/controllers/admin/surveyaction.php,149] 
[D,application/controllers/admin/surveyaction.php,149]         //echo $editsurvey;
[D,application/controllers/admin/surveyaction.php,150]         //$this->load->model('questions_model');
[C,application/controllers/admin/surveyaction.php,150]         $oResult = Questions::model()->getQuestionsWithSubQuestions($surveyid, $esrow['language'], "({{questions}}.type = 'T'  OR  {{questions}}.type = 'Q'  OR  {{questions}}.type = 'T' OR {{questions}}.type = 'S')");
[C,application/controllers/admin/surveyaction.php,151] 
[C,application/controllers/admin/surveyaction.php,152]         $aData['questions'] = $oResult;
@@ -158,43 +157,36 @@ function editsurveysettings($surveyid)
[C,application/controllers/admin/surveyaction.php,157]     }
[C,application/controllers/admin/surveyaction.php,158] 
[C,application/controllers/admin/surveyaction.php,159]     /**
[D,application/controllers/admin/surveyaction.php,161]      * survey::importsurveyresources()
[D,application/controllers/admin/surveyaction.php,162]      * Function responsible to import survey resources.
[A,application/controllers/admin/surveyaction.php,160]      * Function responsible to import survey resources from a '.zip' file.
[A,application/controllers/admin/surveyaction.php,161]      *
[C,application/controllers/admin/surveyaction.php,162]      * @access public
[C,application/controllers/admin/surveyaction.php,163]      * @return void
[C,application/controllers/admin/surveyaction.php,164]      */
[C,application/controllers/admin/surveyaction.php,165]     function importsurveyresources()
[C,application/controllers/admin/surveyaction.php,166]     {
[C,application/controllers/admin/surveyaction.php,167]         $clang = $this->getController()->lang;
[D,application/controllers/admin/surveyaction.php,169]         $action = Yii::app()->request->getPost('action');
[D,application/controllers/admin/surveyaction.php,170]         $surveyid = Yii::app()->request->getPost('sid');
[D,application/controllers/admin/surveyaction.php,171]         $aViewUrls = array();
[D,application/controllers/admin/surveyaction.php,172] 
[D,application/controllers/admin/surveyaction.php,173]         $aData = array(
[D,application/controllers/admin/surveyaction.php,174]             'surveyid' => $surveyid,
[D,application/controllers/admin/surveyaction.php,175]             'clang' => $clang
[D,application/controllers/admin/surveyaction.php,176]         );
[D,application/controllers/admin/surveyaction.php,177] 
[D,application/controllers/admin/surveyaction.php,178]         $aData['display']['menu_bars']['surveysummary'] = $action;
[A,application/controllers/admin/surveyaction.php,168]         $surveyid = Yii::app()->request->getPost('surveyid');
[C,application/controllers/admin/surveyaction.php,169] 
[D,application/controllers/admin/surveyaction.php,180]         if ($action == "importsurveyresources" && $surveyid)
[A,application/controllers/admin/surveyaction.php,170]         if (!empty($surveyid))
[C,application/controllers/admin/surveyaction.php,171]         {
[D,application/controllers/admin/surveyaction.php,182]             if (Yii::app()->getConfig('demoMode'))
[D,application/controllers/admin/surveyaction.php,183]                 $this->getController()->error($clang->gT("Demo Mode Only: Uploading file is disabled in this system."));
[D,application/controllers/admin/surveyaction.php,184] 
[D,application/controllers/admin/surveyaction.php,185]             Yii::app()->loadLibrary('admin/Phpzip');
[A,application/controllers/admin/surveyaction.php,172]             $aData['display']['menu_bars']['surveysummary'] = 'importsurveyresources';
[C,application/controllers/admin/surveyaction.php,173] 
[D,application/controllers/admin/surveyaction.php,187]             $zipfile = $_FILES['the_file']['tmp_name'];
[D,application/controllers/admin/surveyaction.php,188]             $z = new PHPZip();
[A,application/controllers/admin/surveyaction.php,174]             if (Yii::app()->getConfig('demoMode'))
[A,application/controllers/admin/surveyaction.php,175]                 $this->getController()->error($clang->gT("Demo mode only: Uploading files is disabled in this system."), $this->getController()->createUrl("admin/survey/view/surveyid/{$surveyid}"));
[C,application/controllers/admin/surveyaction.php,176] 
[C,application/controllers/admin/surveyaction.php,177]             // Create temporary directory
[D,application/controllers/admin/surveyaction.php,191]             // If dangerous content is unzipped then no one will know the path
[D,application/controllers/admin/surveyaction.php,192]             $extractdir = $this->_tempdir(Yii::app()->getConfig('tempdir'));
[D,application/controllers/admin/surveyaction.php,193]             $basedestdir = Yii::app()->getConfig('publicdir') . "/upload/surveys";
[A,application/controllers/admin/surveyaction.php,178]             // If dangerous content is unzipped
[A,application/controllers/admin/surveyaction.php,179]             // then no one will know the path
[A,application/controllers/admin/surveyaction.php,180]             $extractdir = self::_tempdir(Yii::app()->getConfig('tempdir'));
[A,application/controllers/admin/surveyaction.php,181]             $zipfilename = $_FILES['the_file']['tmp_name'];
[A,application/controllers/admin/surveyaction.php,182]             $basedestdir = Yii::app()->getConfig('uploaddir') . "/surveys";
[C,application/controllers/admin/surveyaction.php,183]             $destdir = $basedestdir . "/$surveyid/";
[C,application/controllers/admin/surveyaction.php,184] 
[A,application/controllers/admin/surveyaction.php,185]             Yii::app()->loadLibrary('admin.pclzip.pclzip');
[A,application/controllers/admin/surveyaction.php,186]             $zip = new PclZip($zipfilename);
[A,application/controllers/admin/surveyaction.php,187] 
[C,application/controllers/admin/surveyaction.php,188]             if (!is_writeable($basedestdir))
[D,application/controllers/admin/surveyaction.php,197]                 $this->getController()->error(sprintf($clang->gT("Incorrect permissions in your %s folder."), $basedestdir));
[A,application/controllers/admin/surveyaction.php,189]                 $this->getController()->error(sprintf($clang->gT("Incorrect permissions in your %s folder."), $basedestdir), $this->getController()->createUrl("admin/survey/view/surveyid/{$surveyid}"));
[C,application/controllers/admin/surveyaction.php,190] 
[C,application/controllers/admin/surveyaction.php,191]             if (!is_dir($destdir))
[C,application/controllers/admin/surveyaction.php,192]                 mkdir($destdir);
@@ -202,75 +194,31 @@ function importsurveyresources()
[C,application/controllers/admin/surveyaction.php,194]             $aImportedFilesInfo = null;
[C,application/controllers/admin/surveyaction.php,195]             $aErrorFilesInfo = null;
[C,application/controllers/admin/surveyaction.php,196] 
[D,application/controllers/admin/surveyaction.php,205]             if (is_file($zipfile))
[A,application/controllers/admin/surveyaction.php,197]             if (is_file($zipfilename))
[C,application/controllers/admin/surveyaction.php,198]             {
[D,application/controllers/admin/surveyaction.php,207]                 if ($z->extract($extractdir, $zipfile) != 'OK')
[D,application/controllers/admin/surveyaction.php,208]                     $this->getController()->error($clang->gT("This file is not a valid ZIP file $CI. Import failed."));
[A,application/controllers/admin/surveyaction.php,199]                 if ($zip->extract($extractdir) <= 0)
[A,application/controllers/admin/surveyaction.php,200]                     $this->getController()->error($clang->gT("This file is not a valid ZIP file archive. Import failed. " . $zip->errorInfo(true)), $this->getController()->createUrl("admin/survey/view/surveyid/{$surveyid}"));
[C,application/controllers/admin/surveyaction.php,201] 
[C,application/controllers/admin/surveyaction.php,202]                 // now read tempdir and copy authorized files only
[D,application/controllers/admin/surveyaction.php,211]                 $dh = opendir($extractdir);
[D,application/controllers/admin/surveyaction.php,212]                 while ($direntry = readdir($dh))
[D,application/controllers/admin/surveyaction.php,213]                 {
[D,application/controllers/admin/surveyaction.php,214]                     if (($direntry != ".") && ($direntry != ".."))
[D,application/controllers/admin/surveyaction.php,215]                     {
[D,application/controllers/admin/surveyaction.php,216]                         if (is_file($extractdir . "/" . $direntry))
[D,application/controllers/admin/surveyaction.php,217]                         {
[D,application/controllers/admin/surveyaction.php,218]                             // is  a file
[D,application/controllers/admin/surveyaction.php,219]                             $extfile = substr(strrchr($direntry, '.'), 1);
[D,application/controllers/admin/surveyaction.php,220]                             if (!(stripos(',' . $allowedresourcesuploads . ',', ',' . $extfile . ',') === false))
[D,application/controllers/admin/surveyaction.php,221]                             {
[D,application/controllers/admin/surveyaction.php,222]                                 // Extension allowed
[D,application/controllers/admin/surveyaction.php,223]                                 if (!copy($extractdir . "/" . $direntry, $destdir . $direntry))
[D,application/controllers/admin/surveyaction.php,224]                                 {
[D,application/controllers/admin/surveyaction.php,225]                                     $aErrorFilesInfo[] = Array(
[D,application/controllers/admin/surveyaction.php,226]                                         "filename" => $direntry,
[D,application/controllers/admin/surveyaction.php,227]                                         "status" => $clang->gT("Copy failed")
[D,application/controllers/admin/surveyaction.php,228]                                     );
[D,application/controllers/admin/surveyaction.php,229]                                     unlink($extractdir . "/" . $direntry);
[D,application/controllers/admin/surveyaction.php,230]                                 }
[D,application/controllers/admin/surveyaction.php,231]                                 else
[D,application/controllers/admin/surveyaction.php,232]                                 {
[D,application/controllers/admin/surveyaction.php,233]                                     $aImportedFilesInfo[] = Array(
[D,application/controllers/admin/surveyaction.php,234]                                         "filename" => $direntry,
[D,application/controllers/admin/surveyaction.php,235]                                         "status" => $clang->gT("OK")
[D,application/controllers/admin/surveyaction.php,236]                                     );
[D,application/controllers/admin/surveyaction.php,237]                                     unlink($extractdir . "/" . $direntry);
[D,application/controllers/admin/surveyaction.php,238]                                 }
[D,application/controllers/admin/surveyaction.php,239]                             }
[D,application/controllers/admin/surveyaction.php,240]                             else
[D,application/controllers/admin/surveyaction.php,241]                             { // Extension forbidden
[D,application/controllers/admin/surveyaction.php,242]                                 $aErrorFilesInfo[] = Array(
[D,application/controllers/admin/surveyaction.php,243]                                     "filename" => $direntry,
[D,application/controllers/admin/surveyaction.php,244]                                     "status" => $clang->gT("Error") . " (" . $clang->gT("Forbidden Extension") . ")"
[D,application/controllers/admin/surveyaction.php,245]                                 );
[D,application/controllers/admin/surveyaction.php,246]                                 unlink($extractdir . "/" . $direntry);
[D,application/controllers/admin/surveyaction.php,247]                             }
[D,application/controllers/admin/surveyaction.php,248]                         }
[D,application/controllers/admin/surveyaction.php,249]                     }
[D,application/controllers/admin/surveyaction.php,250]                 }
[A,application/controllers/admin/surveyaction.php,203]                 list($aImportedFilesInfo, $aErrorFilesInfo) = $this->_filterImportedResources($extractdir, $destdir);
[C,application/controllers/admin/surveyaction.php,204] 
[C,application/controllers/admin/surveyaction.php,205]                 // Delete the temporary file
[D,application/controllers/admin/surveyaction.php,253]                 unlink($zipfile);
[D,application/controllers/admin/surveyaction.php,254]                 // Delete temporary folder
[D,application/controllers/admin/surveyaction.php,255]                 rmdir($extractdir);
[A,application/controllers/admin/surveyaction.php,206]                 unlink($zipfilename);
[C,application/controllers/admin/surveyaction.php,207] 
[D,application/controllers/admin/surveyaction.php,257]                 // display summary
[C,application/controllers/admin/surveyaction.php,208]                 if (is_null($aErrorFilesInfo) && is_null($aImportedFilesInfo))
[D,application/controllers/admin/surveyaction.php,259]                     $this->getController()->error(
[D,application/controllers/admin/surveyaction.php,260]                          $clang->gT("This ZIP $CI contains no valid Resources files. Import failed.") . '<br />' .
[D,application/controllers/admin/surveyaction.php,261]                          $clang->gT("Remember that we do not support subdirectories in ZIP $CIs.")
[D,application/controllers/admin/surveyaction.php,262]                     );
[D,application/controllers/admin/surveyaction.php,263] 
[D,application/controllers/admin/surveyaction.php,264]                 $aData['aErrorFilesInfo'] = $aErrorFilesInfo;
[D,application/controllers/admin/surveyaction.php,265]                 $aData['aImportedFilesInfo'] = $aImportedFilesInfo;
[A,application/controllers/admin/surveyaction.php,209]                     $this->getController()->error($clang->gT("This ZIP archive contains no valid Resources files. Import failed."), $this->getController()->createUrl("admin/survey/view/surveyid/{$surveyid}"));
[C,application/controllers/admin/surveyaction.php,210]             }
[C,application/controllers/admin/surveyaction.php,211]             else
[D,application/controllers/admin/surveyaction.php,268]                 $this->getController()->error(sprintf($clang->gT("An error occurred uploading your file. This may be caused by incorrect permissions in your %s folder."), $basedestdir));
[A,application/controllers/admin/surveyaction.php,212]                 $this->getController()->error(sprintf($clang->gT("An error occurred uploading your file. This may be caused by incorrect permissions in your %s folder."), $basedestdir), $this->getController()->createUrl("admin/survey/view/surveyid/{$surveyid}"));
[C,application/controllers/admin/surveyaction.php,213] 
[D,application/controllers/admin/surveyaction.php,270]             $aViewUrls[] = 'importSurveyResources_view';
[D,application/controllers/admin/surveyaction.php,271]         }
[A,application/controllers/admin/surveyaction.php,214]             $aData = array(
[A,application/controllers/admin/surveyaction.php,215]                 'aErrorFilesInfo' => $aErrorFilesInfo,
[A,application/controllers/admin/surveyaction.php,216]                 'aImportedFilesInfo' => $aImportedFilesInfo,
[A,application/controllers/admin/surveyaction.php,217]                 'surveyid' => $surveyid
[A,application/controllers/admin/surveyaction.php,218]             );
[C,application/controllers/admin/surveyaction.php,219] 
[D,application/controllers/admin/surveyaction.php,273]         $this->_renderWrappedTemplate($aViewUrls, $aData);
[A,application/controllers/admin/surveyaction.php,220]             $this->_renderWrappedTemplate('importSurveyResources_view', $aData);
[A,application/controllers/admin/surveyaction.php,221]         }
[C,application/controllers/admin/surveyaction.php,222]     }
[C,application/controllers/admin/surveyaction.php,223] 
[C,application/controllers/admin/surveyaction.php,224]     /**
@@ -917,4 +917,64 @@ function _userGroupBar($ugid = 0)
[C,application/core/Survey_Common_Action.php,917]         $this->getController()->render('/admin/usergroup/usergroupbar_view', $data);
[C,application/core/Survey_Common_Action.php,918]     }
[C,application/core/Survey_Common_Action.php,919] 
[A,application/core/Survey_Common_Action.php,920]     protected function _filterImportedResources($extractdir, $destdir)
[A,application/core/Survey_Common_Action.php,921]     {
[A,application/core/Survey_Common_Action.php,922]         $clang = $this->getController()->lang;
[A,application/core/Survey_Common_Action.php,923]         $dh = opendir($extractdir);
[A,application/core/Survey_Common_Action.php,924]         $aErrorFilesInfo = array();
[A,application/core/Survey_Common_Action.php,925]         $aImportedFilesInfo = array();
[A,application/core/Survey_Common_Action.php,926] 
[A,application/core/Survey_Common_Action.php,927]         if (!is_dir($destdir))
[A,application/core/Survey_Common_Action.php,928]             mkdir($destdir);
[A,application/core/Survey_Common_Action.php,929] 
[A,application/core/Survey_Common_Action.php,930]         while ($direntry = readdir($dh))
[A,application/core/Survey_Common_Action.php,931]         {
[A,application/core/Survey_Common_Action.php,932]             if ($direntry != "." && $direntry != "..")
[A,application/core/Survey_Common_Action.php,933]             {
[A,application/core/Survey_Common_Action.php,934]                 if (is_file($extractdir . "/" . $direntry))
[A,application/core/Survey_Common_Action.php,935]                 {
[A,application/core/Survey_Common_Action.php,936]                     // is  a file
[A,application/core/Survey_Common_Action.php,937]                     $extfile = substr(strrchr($direntry, '.'), 1);
[A,application/core/Survey_Common_Action.php,938]                     if (!(stripos(',' . Yii::app()->getConfig('allowedresourcesuploads') . ',', ',' . $extfile . ',') === false))
[A,application/core/Survey_Common_Action.php,939]                     {
[A,application/core/Survey_Common_Action.php,940]                         // Extension allowed
[A,application/core/Survey_Common_Action.php,941]                         if (!copy($extractdir . "/" . $direntry, $destdir . "/" . $direntry))
[A,application/core/Survey_Common_Action.php,942]                         {
[A,application/core/Survey_Common_Action.php,943]                             $aErrorFilesInfo[] = Array(
[A,application/core/Survey_Common_Action.php,944]                                 "filename" => $direntry,
[A,application/core/Survey_Common_Action.php,945]                                 "status" => $clang->gT("Copy failed")
[A,application/core/Survey_Common_Action.php,946]                             );
[A,application/core/Survey_Common_Action.php,947]                         }
[A,application/core/Survey_Common_Action.php,948]                         else
[A,application/core/Survey_Common_Action.php,949]                         {
[A,application/core/Survey_Common_Action.php,950]                             $aImportedFilesInfo[] = Array(
[A,application/core/Survey_Common_Action.php,951]                                 "filename" => $direntry,
[A,application/core/Survey_Common_Action.php,952]                                 "status" => $clang->gT("OK")
[A,application/core/Survey_Common_Action.php,953]                             );
[A,application/core/Survey_Common_Action.php,954]                         }
[A,application/core/Survey_Common_Action.php,955]                     }
[A,application/core/Survey_Common_Action.php,956]                     else
[A,application/core/Survey_Common_Action.php,957]                     {
[A,application/core/Survey_Common_Action.php,958]                         // Extension forbidden
[A,application/core/Survey_Common_Action.php,959]                         $aErrorFilesInfo[] = Array(
[A,application/core/Survey_Common_Action.php,960]                             "filename" => $direntry,
[A,application/core/Survey_Common_Action.php,961]                             "status" => $clang->gT("Forbidden Extension")
[A,application/core/Survey_Common_Action.php,962]                         );
[A,application/core/Survey_Common_Action.php,963]                     }
[A,application/core/Survey_Common_Action.php,964]                     unlink($extractdir . "/" . $direntry);
[A,application/core/Survey_Common_Action.php,965]                 }
[A,application/core/Survey_Common_Action.php,966]                 elseif (is_dir($extractdir . "/" . $direntry))
[A,application/core/Survey_Common_Action.php,967]                 {
[A,application/core/Survey_Common_Action.php,968]                     list($_aImportedFilesInfo, $_aErrorFilesInfo) = $this->_filterImportedFiles($extractdir . "/" . $direntry, $destdir . "/" . $direntry);
[A,application/core/Survey_Common_Action.php,969]                     $aImportedFilesInfo = array_merge($aImportedFilesInfo, $_aImportedFilesInfo);
[A,application/core/Survey_Common_Action.php,970]                     $aErrorFilesInfo = array_merge($aErrorFilesInfo, $_aErrorFilesInfo);
[A,application/core/Survey_Common_Action.php,971]                 }
[A,application/core/Survey_Common_Action.php,972]             }
[A,application/core/Survey_Common_Action.php,973]         }
[A,application/core/Survey_Common_Action.php,974] 
[A,application/core/Survey_Common_Action.php,975]         rmdir($extractdir);
[A,application/core/Survey_Common_Action.php,976] 
[A,application/core/Survey_Common_Action.php,977]         return array($aImportedFilesInfo, $aErrorFilesInfo);
[A,application/core/Survey_Common_Action.php,978]     }
[A,application/core/Survey_Common_Action.php,979] 
[C,application/core/Survey_Common_Action.php,980] }
@@ -137,7 +137,7 @@ function modlabelsetanswers($lid)
[C,application/helpers/admin/label_helper.php,137]         $_POST['method'] = $clang->gT("Save");
[C,application/helpers/admin/label_helper.php,138]     }
[C,application/helpers/admin/label_helper.php,139] 
[D,application/helpers/admin/label_helper.php,140]     $data = json_decode(stripslashes($_POST['dataToSend']));
[A,application/helpers/admin/label_helper.php,140]     $data = json_decode(CHttpRequest::getPost('dataToSend'));
[C,application/helpers/admin/label_helper.php,141] 
[C,application/helpers/admin/label_helper.php,142]     if ($ajax)
[C,application/helpers/admin/label_helper.php,143]         $lid = insertlabelset();
@@ -145,7 +145,7 @@ function modlabelsetanswers($lid)
[C,application/helpers/admin/label_helper.php,145]     if (count(array_unique($data->{'codelist'})) == count($data->{'codelist'}))
[C,application/helpers/admin/label_helper.php,146]     {
[C,application/helpers/admin/label_helper.php,147] 
[D,application/helpers/admin/label_helper.php,148]         $query = "DELETE FROM {{labels}}  WHERE `lid` = '$lid'";
[A,application/helpers/admin/label_helper.php,148]         $query = "DELETE FROM {{labels}} WHERE `lid` = '$lid'";
[C,application/helpers/admin/label_helper.php,149] 
[C,application/helpers/admin/label_helper.php,150]         $result = Yii::app()->db->createCommand($query)->execute();
[C,application/helpers/admin/label_helper.php,151] 
@@ -30,5 +30,5 @@
[C,application/views/admin/labels/import_view.php,30]         <li><?php echo $clang->gT("Labels") . ": {$aImportResults['labels']}" ?></li>
[C,application/views/admin/labels/import_view.php,31]     </ul>
[C,application/views/admin/labels/import_view.php,32]     <strong><?php $clang->eT("Import of label set(s) is completed.") ?></strong><br /><br />
[D,application/views/admin/labels/import_view.php,33]     <input type='submit' value='<?php $clang->gT("Return to label set administration") ?>' onclick="window.open('<?php echo $this->createUrl('admin/labels/view') ?>', '_top')" />
[A,application/views/admin/labels/import_view.php,33]     <input type='submit' value='<?php $clang->eT("Return to label set administration"); ?>' onclick="window.open('<?php echo $this->createUrl('admin/labels/view') ?>', '_top')" />
[C,application/views/admin/labels/import_view.php,34] </div><br />
@@ -1,4 +1,4 @@
[D,application/views/admin/labels/importlabelresources_view.php,1] <div class='header ui-widget-header'><?php $clang->eT("Import Label Set") ?></div>
[A,application/views/admin/labels/importlabelresources_view.php,1] <div class='header ui-widget-header'><?php $clang->eT("Import Label Set Resources") ?></div>
[C,application/views/admin/labels/importlabelresources_view.php,2] <div class='messagebox ui-corner-all'>
[C,application/views/admin/labels/importlabelresources_view.php,3]     <div class="successheader"><?php $clang->eT("Success") ?></div><br />
[C,application/views/admin/labels/importlabelresources_view.php,4]     <?php $clang->eT("File upload succeeded.") ?><br /><br />
@@ -124,13 +124,13 @@
[C,application/views/admin/labels/labelview_view.php,124]                         <li>
[C,application/views/admin/labels/labelview_view.php,125]                             <label>&nbsp;</label>
[C,application/views/admin/labels/labelview_view.php,126]                             <input type='button'<?php echo hasResources($lid, 'label') === false ? ' disabled="disabled"' : '' ?>
[D,application/views/admin/labels/labelview_view.php,127]                                    onclick='window.open("<?php echo Yii::app()->getConfig('sCKEditorURL') ?>/editor/filemanager/browser/default/browser.html?Connector=../../connectors/php/connector.php?", "_blank")'
[A,application/views/admin/labels/labelview_view.php,127]                                    onclick='window.open("<?php echo $this->createUrl("admin/kcfinder/index/load/browse"); ?>", "_blank")'
[C,application/views/admin/labels/labelview_view.php,128]                                    value="<?php $clang->eT("Browse Uploaded Resources") ?>" />
[C,application/views/admin/labels/labelview_view.php,129]                         </li>
[C,application/views/admin/labels/labelview_view.php,130]                         <li>
[C,application/views/admin/labels/labelview_view.php,131]                             <label>&nbsp;</label>
[C,application/views/admin/labels/labelview_view.php,132]                             <input type='button'<?php echo hasResources($lid, 'label') === false ? ' disabled="disabled"' : '' ?>
[D,application/views/admin/labels/labelview_view.php,133]                                    onclick='window.open("<?php echo $this->createUrl("/admin/labels/exportlabelresources/lid/$lid"); ?>", "_blank")'
[A,application/views/admin/labels/labelview_view.php,133]                                    onclick='window.open("<?php echo $this->createUrl("/admin/export/resources/export/label/lid/$lid"); ?>", "_blank")'
[C,application/views/admin/labels/labelview_view.php,134]                                    value="<?php $clang->eT("Export Resources As ZIP Archive") ?>"  />
[C,application/views/admin/labels/labelview_view.php,135]                         </li>
[C,application/views/admin/labels/labelview_view.php,136]                         <li>
@@ -8,7 +8,7 @@
[C,application/views/admin/survey/deleteSurvey_view.php,8]         <font color='red'><strong><?php $clang->eT("Error"); ?></strong></font>
[C,application/views/admin/survey/deleteSurvey_view.php,9]         <?php $clang->eT("You have not selected a survey to delete"); ?>
[C,application/views/admin/survey/deleteSurvey_view.php,10]         <br /><br />
[D,application/views/admin/survey/deleteSurvey_view.php,11]         <input type='submit' value='<?php $clang->eT("Main Admin Screen"); ?>' onclick="window.open('<?php echo Yii::app()->getController()->createUrl("admin"); ?>', '_top')"/>
[A,application/views/admin/survey/deleteSurvey_view.php,11]         <input type='submit' value='<?php $clang->eT("Main Admin Screen"); ?>' onclick="window.open('<?php echo $this->createUrl('admin/index'); ?>', '_top')"/>
[C,application/views/admin/survey/deleteSurvey_view.php,12]         </td></tr></table>
[C,application/views/admin/survey/deleteSurvey_view.php,13]         </body></html>
[C,application/views/admin/survey/deleteSurvey_view.php,14]         <?php return;
@@ -21,18 +21,18 @@
[C,application/views/admin/survey/deleteSurvey_view.php,21]         <?php $clang->eT("It will also delete any resources/files that have been uploaded for this survey."); ?><br /><br />
[C,application/views/admin/survey/deleteSurvey_view.php,22]         <?php $clang->eT("We recommend that before you delete this survey you export the entire survey from the main administration screen.");
[C,application/views/admin/survey/deleteSurvey_view.php,23] 
[D,application/views/admin/survey/deleteSurvey_view.php,24]         if (Yii::app()->db->schema->getTable('{{survey_$surveyid}}'))
[A,application/views/admin/survey/deleteSurvey_view.php,24]         if (tableExists("{{survey_{$surveyid}}}"))
[C,application/views/admin/survey/deleteSurvey_view.php,25]         { ?>
[C,application/views/admin/survey/deleteSurvey_view.php,26]             <br /><br /><?php $clang->eT("This survey is active and a responses table exists. If you delete this survey, these responses (and files) will be deleted. We recommend that you export the responses before deleting this survey."); ?><br /><br />
[C,application/views/admin/survey/deleteSurvey_view.php,27]         <?php }
[C,application/views/admin/survey/deleteSurvey_view.php,28] 
[D,application/views/admin/survey/deleteSurvey_view.php,29]         if (Yii::app()->db->schema->getTable('{{tokens_$surveyid}}'))
[A,application/views/admin/survey/deleteSurvey_view.php,29]         if (tableExists("{{tokens_{$surveyid}}}"))
[C,application/views/admin/survey/deleteSurvey_view.php,30]         { ?>
[C,application/views/admin/survey/deleteSurvey_view.php,31]             <?php $clang->eT("This survey has an associated tokens table. If you delete this survey this tokens table will be deleted. We recommend that you export or backup these tokens before deleting this survey."); ?><br /><br />
[C,application/views/admin/survey/deleteSurvey_view.php,32]         <?php } ?>
[C,application/views/admin/survey/deleteSurvey_view.php,33] 
[C,application/views/admin/survey/deleteSurvey_view.php,34]         <p>
[D,application/views/admin/survey/deleteSurvey_view.php,35]         <input type='submit'  value='<?php $clang->eT("Delete survey"); ?>' onclick="window.open('<?php echo Yii::app()->getController()->createUrl("admin/survey/delete/surveyid/$surveyid"); ?>', '_top')" />
[D,application/views/admin/survey/deleteSurvey_view.php,36]         <input type='submit'  value='<?php $clang->eT("Cancel"); ?>' onclick="window.open('<?php echo Yii::app()->getController()->createUrl("admin/survey/view/$surveyid"); ?>', '_top')" />
[A,application/views/admin/survey/deleteSurvey_view.php,35]         <input type='submit' value='<?php $clang->eT("Delete survey"); ?>' onclick="window.open('<?php echo $this->createUrl("admin/survey/delete/surveyid/$surveyid/delete/yes"); ?>', '_top')" />
[A,application/views/admin/survey/deleteSurvey_view.php,36]         <input type='submit' value='<?php $clang->eT("Cancel"); ?>' onclick="window.open('<?php echo $this->createUrl("admin/survey/view/$surveyid"); ?>', '_top')" />
[C,application/views/admin/survey/deleteSurvey_view.php,37]     <?php } ?>
[C,application/views/admin/survey/deleteSurvey_view.php,38] </div><br />&nbsp;
@@ -1,24 +1,52 @@
[D,application/views/admin/survey/importSurveyResources_view.php,1] <div class='header ui-widget-header'><?php echo $clang->gT("Import survey resources"); ?></div>
[A,application/views/admin/survey/importSurveyResources_view.php,1] <div class='header ui-widget-header'><?php $clang->eT("Import survey resources"); ?></div>
[C,application/views/admin/survey/importSurveyResources_view.php,2] <div class='messagebox ui-corner-all'>
[D,application/views/admin/survey/importSurveyResources_view.php,3]     <strong><?php echo $clang->gT("Imported Resources for"); ?>" SID:</strong> <?php echo $surveyid; ?><br /><br />
[A,application/views/admin/survey/importSurveyResources_view.php,3]     <div class="successheader"><?php $clang->eT("Success") ?></div><br />
[A,application/views/admin/survey/importSurveyResources_view.php,4]     <?php $clang->eT("File upload succeeded.") ?><br /><br />
[A,application/views/admin/survey/importSurveyResources_view.php,5]     <?php $clang->eT("Reading file..") ?><br /><br />
[A,application/views/admin/survey/importSurveyResources_view.php,6] <?php
[A,application/views/admin/survey/importSurveyResources_view.php,7]     $ImportListHeader = '';
[A,application/views/admin/survey/importSurveyResources_view.php,8]     if (!count($aErrorFilesInfo) &&count($aImportedFilesInfo))
[A,application/views/admin/survey/importSurveyResources_view.php,9]     {
[A,application/views/admin/survey/importSurveyResources_view.php,10]         $status = $clang->gT("Success");
[A,application/views/admin/survey/importSurveyResources_view.php,11]         $statusClass = 'successheader';
[A,application/views/admin/survey/importSurveyResources_view.php,12]         $okfiles = count($aImportedFilesInfo);
[A,application/views/admin/survey/importSurveyResources_view.php,13]         $errfiles=0;
[A,application/views/admin/survey/importSurveyResources_view.php,14]         $ImportListHeader .= "<br /><strong><u>" . $clang->gT("Imported Files List") . ":</u></strong><br />\n";
[A,application/views/admin/survey/importSurveyResources_view.php,15]     }
[A,application/views/admin/survey/importSurveyResources_view.php,16]     elseif (count($aErrorFilesInfo) &&count($aImportedFilesInfo))
[A,application/views/admin/survey/importSurveyResources_view.php,17]     {
[A,application/views/admin/survey/importSurveyResources_view.php,18]         $status = $clang->gT("Partial");
[A,application/views/admin/survey/importSurveyResources_view.php,19]         $statusClass = 'partialheader';
[A,application/views/admin/survey/importSurveyResources_view.php,20]         $okfiles = count($aImportedFilesInfo);
[A,application/views/admin/survey/importSurveyResources_view.php,21]         $errfiles = count($aErrorFilesInfo);
[A,application/views/admin/survey/importSurveyResources_view.php,22]         $ErrorListHeader = "<br /><strong><u>" . $clang->gT("Error Files List") . ":</u></strong><br />\n";
[A,application/views/admin/survey/importSurveyResources_view.php,23]         $ImportListHeader .= "<br /><strong><u>" . $clang->gT("Imported Files List") . ":</u></strong><br />\n";
[A,application/views/admin/survey/importSurveyResources_view.php,24]     }
[A,application/views/admin/survey/importSurveyResources_view.php,25]     else
[A,application/views/admin/survey/importSurveyResources_view.php,26]     {
[A,application/views/admin/survey/importSurveyResources_view.php,27]         $okfiles = 0;
[A,application/views/admin/survey/importSurveyResources_view.php,28]         $status = $clang->gT("Error");
[A,application/views/admin/survey/importSurveyResources_view.php,29]         $statusClass = 'warningheader';
[A,application/views/admin/survey/importSurveyResources_view.php,30]         $errfiles = count($aErrorFilesInfo);
[A,application/views/admin/survey/importSurveyResources_view.php,31]         $ErrorListHeader = "<br /><strong><u>" . $clang->gT("Error Files List") . ":</u></strong><br />\n";
[A,application/views/admin/survey/importSurveyResources_view.php,32]     }
[A,application/views/admin/survey/importSurveyResources_view.php,33] ?>
[A,application/views/admin/survey/importSurveyResources_view.php,34]     <strong><?php $clang->eT("Imported Resources for"); ?>" SID:</strong> <?php echo $surveyid; ?><br /><br />
[C,application/views/admin/survey/importSurveyResources_view.php,35]     <div class="<?php echo $statusClass; ?>">
[C,application/views/admin/survey/importSurveyResources_view.php,36]         <?php echo $status; ?>
[C,application/views/admin/survey/importSurveyResources_view.php,37]     </div>
[C,application/views/admin/survey/importSurveyResources_view.php,38]     <br />
[C,application/views/admin/survey/importSurveyResources_view.php,39]     <strong>
[D,application/views/admin/survey/importSurveyResources_view.php,9]         <u><?php echo $clang->gT("Resources Import Summary"); ?></u>
[A,application/views/admin/survey/importSurveyResources_view.php,40]         <u><?php $clang->eT("Resources Import Summary"); ?></u>
[C,application/views/admin/survey/importSurveyResources_view.php,41]     </strong>
[C,application/views/admin/survey/importSurveyResources_view.php,42]     <br />
[D,application/views/admin/survey/importSurveyResources_view.php,12]     <?php echo $clang->gT("Total Imported files"); ?>: <?php echo count($aImportedFilesInfo); ?><br />
[D,application/views/admin/survey/importSurveyResources_view.php,13]     <?php echo $clang->gT("Total Errors"); ?>: <?php echo $count($aErrorFilesInfo); ?><br />
[D,application/views/admin/survey/importSurveyResources_view.php,14]     <div class="successheader"><?php echo !empty($aImportedFilesInfo) && empty($aErrorFilesInfo)? $clang->gT("Success") : $clang->gT("Partial"); ?></div><br /><br />
[D,application/views/admin/survey/importSurveyResources_view.php,15]     <?php echo $clang->gT("File upload succeeded.") ?><br /><br />
[D,application/views/admin/survey/importSurveyResources_view.php,16]     <?php echo $clang->gT("Reading file...") ?><br /><br />
[D,application/views/admin/survey/importSurveyResources_view.php,17] <?php
[A,application/views/admin/survey/importSurveyResources_view.php,43]     <?php $clang->eT("Total Imported files"); ?>: <?php echo count($okfiles); ?><br />
[A,application/views/admin/survey/importSurveyResources_view.php,44]     <?php $clang->eT("Total Errors"); ?>: <?php echo count($errfiles); ?><br />
[A,application/views/admin/survey/importSurveyResources_view.php,45]     <?php
[C,application/views/admin/survey/importSurveyResources_view.php,46] if (!empty($aImportedFilesInfo))
[C,application/views/admin/survey/importSurveyResources_view.php,47] {
[C,application/views/admin/survey/importSurveyResources_view.php,48] ?>
[D,application/views/admin/survey/importSurveyResources_view.php,21]     <strong><?php echo $clang->gT("Imported Files List") ?>:</strong><br />
[A,application/views/admin/survey/importSurveyResources_view.php,49]     <strong><?php $clang->eT("Imported Files List") ?>:</strong><br />
[C,application/views/admin/survey/importSurveyResources_view.php,50]     <ul>
[C,application/views/admin/survey/importSurveyResources_view.php,51] <?php
[C,application/views/admin/survey/importSurveyResources_view.php,52]     foreach ($aImportedFilesInfo as $entry)
@@ -33,18 +61,15 @@
[C,application/views/admin/survey/importSurveyResources_view.php,61] <?php
[C,application/views/admin/survey/importSurveyResources_view.php,62] if (!empty($aErrorFilesInfo))
[C,application/views/admin/survey/importSurveyResources_view.php,63] {
[D,application/views/admin/survey/importSurveyResources_view.php,36] ?>
[D,application/views/admin/survey/importSurveyResources_view.php,37]     <strong><?php echo $clang->gT("Error Files List") ?>:</strong><br />
[D,application/views/admin/survey/importSurveyResources_view.php,38]     <ul>
[D,application/views/admin/survey/importSurveyResources_view.php,39] <?php
[A,application/views/admin/survey/importSurveyResources_view.php,64]     echo $ErrorListHeader;
[C,application/views/admin/survey/importSurveyResources_view.php,65]     foreach ($aErrorFilesInfo as $entry)
[C,application/views/admin/survey/importSurveyResources_view.php,66]     {
[D,application/views/admin/survey/importSurveyResources_view.php,42]         echo CHtml::tag('li', array(), $clang->gT("File") . ': ' . $entry["filename"]);
[A,application/views/admin/survey/importSurveyResources_view.php,67]         echo CHtml::tag('li', array(), $clang->gT("File") . ': ' . $entry['filename'] . " (" . $entry['status'] . ")");
[C,application/views/admin/survey/importSurveyResources_view.php,68]     }
[C,application/views/admin/survey/importSurveyResources_view.php,69] ?>
[C,application/views/admin/survey/importSurveyResources_view.php,70]     </ul>
[C,application/views/admin/survey/importSurveyResources_view.php,71] <?php
[C,application/views/admin/survey/importSurveyResources_view.php,72] }
[C,application/views/admin/survey/importSurveyResources_view.php,73] ?>
[D,application/views/admin/survey/importSurveyResources_view.php,49]     <input type='submit' value='<?php echo $clang->gT("Back"); ?>' onclick="window.open('<?php echo $this->createUrl('admin/survey/editsurveysettings/surveyid/' . $surveyid); ?>', '_top')" />\n";
[A,application/views/admin/survey/importSurveyResources_view.php,74]     <input type='submit' value='<?php $clang->eT("Back"); ?>' onclick="window.open('<?php echo $this->createUrl('admin/survey/editsurveysettings/surveyid/' . $surveyid); ?>', '_top')" />
[C,application/views/admin/survey/importSurveyResources_view.php,75] </div>
@@ -1,21 +1,16 @@
[D,application/views/admin/survey/subview/tabResourceManagement_view.php,1] <?php
[D,application/views/admin/survey/subview/tabResourceManagement_view.php,2] 	$yii = Yii::app();
[D,application/views/admin/survey/subview/tabResourceManagement_view.php,3] 	$controller = $yii->getController();
[D,application/views/admin/survey/subview/tabResourceManagement_view.php,4] ?>
[C,application/views/admin/survey/subview/tabResourceManagement_view.php,1] <div id='resources'>
[D,application/views/admin/survey/subview/tabResourceManagement_view.php,6]             <form enctype='multipart/form-data'  class='form30' id='importsurveyresources' name='importsurveyresources' action='<?php echo $yii->homeUrl.('/admin/survey/importsurveyresources/'); ?>' method='post' onsubmit='return validatefilename(this,"<?php $clang->eT('Please select a file to import!','js'); ?>");'>
[D,application/views/admin/survey/subview/tabResourceManagement_view.php,7]             <input type='hidden' name='sid' value='<?php echo $surveyid; ?>' />
[D,application/views/admin/survey/subview/tabResourceManagement_view.php,8]             <input type='hidden' name='action' value='importsurveyresources' />
[D,application/views/admin/survey/subview/tabResourceManagement_view.php,9]             <ul>
[A,application/views/admin/survey/subview/tabResourceManagement_view.php,2]     <form enctype='multipart/form-data'  class='form30' id='importsurveyresources' name='importsurveyresources' action='<?php echo $this->createUrl('admin/survey/importsurveyresources/'); ?>' method='post' onsubmit='return validatefilename(this,"<?php $clang->eT('Please select a file to import!', 'js'); ?>");'>
[A,application/views/admin/survey/subview/tabResourceManagement_view.php,3]         <input type='hidden' name='surveyid' value='<?php echo $surveyid; ?>' />
[A,application/views/admin/survey/subview/tabResourceManagement_view.php,4]         <input type='hidden' name='action' value='importsurveyresources' />
[A,application/views/admin/survey/subview/tabResourceManagement_view.php,5]         <ul>
[C,application/views/admin/survey/subview/tabResourceManagement_view.php,6]             <li><label>&nbsp;</label>
[D,application/views/admin/survey/subview/tabResourceManagement_view.php,11]             <input type='button' onclick='window.open("<?php echo $yii->getConfig('sCKEditorURL'); ?>/editor/filemanager/browser/default/browser.html?Connector=../../connectors/php/connector.php", "_blank")' value="<?php $clang->eT("Browse Uploaded Resources"); ?>" <?php echo $disabledIfNoResources; ?> /></li>
[A,application/views/admin/survey/subview/tabResourceManagement_view.php,7]                 <input type='button' onclick='window.open("<?php echo $this->createUrl("admin/kcfinder/index/load/browse"); ?>", "_blank")' value="<?php $clang->eT("Browse Uploaded Resources"); ?>" <?php echo $disabledIfNoResources; ?> /></li>
[C,application/views/admin/survey/subview/tabResourceManagement_view.php,8]             <li><label>&nbsp;</label>
[D,application/views/admin/survey/subview/tabResourceManagement_view.php,13]             <input type='button' onclick='window.open("<?php echo $yii->homeUrl.("/admin/export/resources/exportsurvresources/surveyid/$surveyid/"); ?>", "_blank")' value="<?php $clang->eT("Export Resources As ZIP Archive"); ?>" <?php echo $disabledIfNoResources; ?> /></li>
[A,application/views/admin/survey/subview/tabResourceManagement_view.php,9]                 <input type='button' onclick='window.open("<?php echo $this->createUrl("admin/export/resources/export/survey/surveyid/$surveyid/"); ?>", "_blank")' value="<?php $clang->eT("Export Resources As ZIP Archive"); ?>" <?php echo $disabledIfNoResources; ?> /></li>
[C,application/views/admin/survey/subview/tabResourceManagement_view.php,10]             <li><label for='the_file'><?php $clang->eT("Select ZIP File:"); ?></label>
[D,application/views/admin/survey/subview/tabResourceManagement_view.php,15]             <input id='the_file' name='the_file' type='file' size='50' /></li>
[A,application/views/admin/survey/subview/tabResourceManagement_view.php,11]                 <input id='the_file' name='the_file' type='file' size='50' /></li>
[C,application/views/admin/survey/subview/tabResourceManagement_view.php,12]             <li><label>&nbsp;</label>
[D,application/views/admin/survey/subview/tabResourceManagement_view.php,17]             <input type='button' value='<?php $clang->eT("Import Resources ZIP Archive"); ?>' <?php echo $ZIPimportAction; ?> /></li>
[D,application/views/admin/survey/subview/tabResourceManagement_view.php,18]             </ul></form>
[D,application/views/admin/survey/subview/tabResourceManagement_view.php,19] 
[D,application/views/admin/survey/subview/tabResourceManagement_view.php,20]         
[A,application/views/admin/survey/subview/tabResourceManagement_view.php,13]                 <input type='button' value='<?php $clang->eT("Import Resources ZIP Archive"); ?>' <?php echo $ZIPimportAction; ?> /></li>
[A,application/views/admin/survey/subview/tabResourceManagement_view.php,14]         </ul>
[A,application/views/admin/survey/subview/tabResourceManagement_view.php,15]     </form>
[C,application/views/admin/survey/subview/tabResourceManagement_view.php,16] </div>
@@ -713,7 +713,7 @@ function js2php(object){
[C,scripts/admin/admin_core.js,713]     for (property in object){
[C,scripts/admin/admin_core.js,714]         var value = object[property];
[C,scripts/admin/admin_core.js,715]         if (typeof(value)=="string"){
[D,scripts/admin/admin_core.js,716]             json += '"'+property+'":"'+value+'",'
[A,scripts/admin/admin_core.js,716]             json += '"'+property+'":"'+value.replace(/[\\"']/g, '\\$&')+'",'
[C,scripts/admin/admin_core.js,717]         }
[C,scripts/admin/admin_core.js,718]         else{
[C,scripts/admin/admin_core.js,719]             if (!value[0]){
@@ -1,13 +1,13 @@
[C,scripts/admin/ckeditor.36/limesurvey-config.js,1] CKEDITOR.editorConfig = function( config )
[C,scripts/admin/ckeditor.36/limesurvey-config.js,2] {
[C,scripts/admin/ckeditor.36/limesurvey-config.js,3] 
[D,scripts/admin/ckeditor.36/limesurvey-config.js,4]     config.filebrowserBrowseUrl = CKEDITOR.basePath+'../kcfinder/browse.php?type=files';
[D,scripts/admin/ckeditor.36/limesurvey-config.js,5]     config.filebrowserImageBrowseUrl = CKEDITOR.basePath+'../kcfinder/browse.php?type=images';
[D,scripts/admin/ckeditor.36/limesurvey-config.js,6]     config.filebrowserFlashBrowseUrl = CKEDITOR.basePath+'../kcfinder/browse.php?type=flash';
[A,scripts/admin/ckeditor.36/limesurvey-config.js,4]     config.filebrowserBrowseUrl = CKEDITOR.basePath+'../../../../admin/kcfinder/index/load/browse?type=files';
[A,scripts/admin/ckeditor.36/limesurvey-config.js,5]     config.filebrowserImageBrowseUrl = CKEDITOR.basePath+'../../../../admin/kcfinder/index/load/browse?type=images';
[A,scripts/admin/ckeditor.36/limesurvey-config.js,6]     config.filebrowserFlashBrowseUrl = CKEDITOR.basePath+'../../../../admin/kcfinder/index/load/browse?type=flash';
[C,scripts/admin/ckeditor.36/limesurvey-config.js,7] 
[D,scripts/admin/ckeditor.36/limesurvey-config.js,8]     config.filebrowserUploadUrl = CKEDITOR.basePath+'../kcfinder/upload.php?type=files';
[D,scripts/admin/ckeditor.36/limesurvey-config.js,9]     config.filebrowserImageUploadUrl = CKEDITOR.basePath+'../kcfinder/upload.php?type=images';
[D,scripts/admin/ckeditor.36/limesurvey-config.js,10]     config.filebrowserFlashUploadUrl = CKEDITOR.basePath+'../kcfinder/upload.php?type=flash';
[A,scripts/admin/ckeditor.36/limesurvey-config.js,8]     config.filebrowserUploadUrl = CKEDITOR.basePath+'../../../../admin/kcfinder/index/load/upload?type=files';
[A,scripts/admin/ckeditor.36/limesurvey-config.js,9]     config.filebrowserImageUploadUrl = CKEDITOR.basePath+'../../../../admin/kcfinder/index/load/upload?type=images';
[A,scripts/admin/ckeditor.36/limesurvey-config.js,10]     config.filebrowserFlashUploadUrl = CKEDITOR.basePath+'../../../../admin/kcfinder/index/load/upload?type=flash';
[C,scripts/admin/ckeditor.36/limesurvey-config.js,11] 
[C,scripts/admin/ckeditor.36/limesurvey-config.js,12] 	config.skin = 'office2003';
[C,scripts/admin/ckeditor.36/limesurvey-config.js,13] 	config.toolbarCanCollapse = false;
@@ -10,85 +10,4 @@
[C,scripts/admin/kcfinder/core/autoload.php,10] *   @license http://www.opensource.org/licenses/gpl-2.0.php GPLv2
[C,scripts/admin/kcfinder/core/autoload.php,11] *   @license http://www.opensource.org/licenses/lgpl-2.1.php LGPLv2
[C,scripts/admin/kcfinder/core/autoload.php,12] *      @link http://kcfinder.sunhater.com
[D,scripts/admin/kcfinder/core/autoload.php,13] */
[D,scripts/admin/kcfinder/core/autoload.php,14] 
[D,scripts/admin/kcfinder/core/autoload.php,15] require_once(dirname(__FILE__).'/../../../../config-defaults.php');
[D,scripts/admin/kcfinder/core/autoload.php,16] require_once(dirname(__FILE__).'/../../../../common.php');
[D,scripts/admin/kcfinder/core/autoload.php,17] require_once(dirname(__FILE__).'/../../../admin_functions.php');
[D,scripts/admin/kcfinder/core/autoload.php,18] 
[D,scripts/admin/kcfinder/core/autoload.php,19] $usquery = "SELECT stg_value FROM ".db_table_name("settings_global")." where stg_name='SessionName'";
[D,scripts/admin/kcfinder/core/autoload.php,20] $usresult = db_execute_assoc($usquery,'',true);
[D,scripts/admin/kcfinder/core/autoload.php,21] if ($usresult)
[D,scripts/admin/kcfinder/core/autoload.php,22] {
[D,scripts/admin/kcfinder/core/autoload.php,23]     $usrow = $usresult->FetchRow();
[D,scripts/admin/kcfinder/core/autoload.php,24]     @session_name($usrow['stg_value']);
[D,scripts/admin/kcfinder/core/autoload.php,25] }
[D,scripts/admin/kcfinder/core/autoload.php,26] else
[D,scripts/admin/kcfinder/core/autoload.php,27] {
[D,scripts/admin/kcfinder/core/autoload.php,28]     session_name("LimeSurveyAdmin");
[D,scripts/admin/kcfinder/core/autoload.php,29] }
[D,scripts/admin/kcfinder/core/autoload.php,30] 
[D,scripts/admin/kcfinder/core/autoload.php,31] session_set_cookie_params(0,$relativeurl.'/');
[D,scripts/admin/kcfinder/core/autoload.php,32] 
[D,scripts/admin/kcfinder/core/autoload.php,33] if (session_id() == "") @session_start();
[D,scripts/admin/kcfinder/core/autoload.php,34] 
[D,scripts/admin/kcfinder/core/autoload.php,35] $_SESSION['KCFINDER'] = array();
[D,scripts/admin/kcfinder/core/autoload.php,36] 
[D,scripts/admin/kcfinder/core/autoload.php,37] $sAllowedExtensions = implode(' ',array_map('trim',explode(',',$allowedresourcesuploads)));
[D,scripts/admin/kcfinder/core/autoload.php,38] $_SESSION['KCFINDER']['types']=array('files'=>$sAllowedExtensions,
[D,scripts/admin/kcfinder/core/autoload.php,39] 'flash'=>$sAllowedExtensions,
[D,scripts/admin/kcfinder/core/autoload.php,40] 'images'=>$sAllowedExtensions);
[D,scripts/admin/kcfinder/core/autoload.php,41] 
[D,scripts/admin/kcfinder/core/autoload.php,42] if ($demoMode === false &&
[D,scripts/admin/kcfinder/core/autoload.php,43] isset($_SESSION['loginID']) &&
[D,scripts/admin/kcfinder/core/autoload.php,44] isset($_SESSION['FileManagerContext']))
[D,scripts/admin/kcfinder/core/autoload.php,45] {
[D,scripts/admin/kcfinder/core/autoload.php,46]     // disable upload at survey creation time
[D,scripts/admin/kcfinder/core/autoload.php,47]     // because we don't know the sid yet
[D,scripts/admin/kcfinder/core/autoload.php,48]     if (preg_match('/^(create|edit):(question|group|answer)/',$_SESSION['FileManagerContext']) != 0 ||
[D,scripts/admin/kcfinder/core/autoload.php,49]     preg_match('/^edit:survey/',$_SESSION['FileManagerContext']) !=0 ||
[D,scripts/admin/kcfinder/core/autoload.php,50]     preg_match('/^edit:assessments/',$_SESSION['FileManagerContext']) !=0 ||
[D,scripts/admin/kcfinder/core/autoload.php,51]     preg_match('/^edit:emailsettings/',$_SESSION['FileManagerContext']) != 0)
[D,scripts/admin/kcfinder/core/autoload.php,52]     {
[D,scripts/admin/kcfinder/core/autoload.php,53]         $contextarray=explode(':',$_SESSION['FileManagerContext'],3);
[D,scripts/admin/kcfinder/core/autoload.php,54]         $surveyid=$contextarray[2];
[D,scripts/admin/kcfinder/core/autoload.php,55] 
[D,scripts/admin/kcfinder/core/autoload.php,56] 
[D,scripts/admin/kcfinder/core/autoload.php,57] 
[D,scripts/admin/kcfinder/core/autoload.php,58]         if(bHasSurveyPermission($surveyid,'surveycontent','update'))
[D,scripts/admin/kcfinder/core/autoload.php,59]         {
[D,scripts/admin/kcfinder/core/autoload.php,60]             $_SESSION['KCFINDER']['disabled'] = false ;
[D,scripts/admin/kcfinder/core/autoload.php,61]             $_SESSION['KCFINDER']['uploadURL'] = "{$relativeurl}/upload/surveys/{$surveyid}/" ;
[D,scripts/admin/kcfinder/core/autoload.php,62]             $_SESSION['KCFINDER']['uploadDir'] = $uploaddir.'/surveys/'.$surveyid;
[D,scripts/admin/kcfinder/core/autoload.php,63]         }
[D,scripts/admin/kcfinder/core/autoload.php,64] 
[D,scripts/admin/kcfinder/core/autoload.php,65]     }
[D,scripts/admin/kcfinder/core/autoload.php,66]     elseif (preg_match('/^edit:label/',$_SESSION['FileManagerContext']) != 0)
[D,scripts/admin/kcfinder/core/autoload.php,67]     {
[D,scripts/admin/kcfinder/core/autoload.php,68]         $contextarray=explode(':',$_SESSION['FileManagerContext'],3);
[D,scripts/admin/kcfinder/core/autoload.php,69]         $labelid=$contextarray[2];
[D,scripts/admin/kcfinder/core/autoload.php,70]         // check if the user has label management right and labelid defined
[D,scripts/admin/kcfinder/core/autoload.php,71]         if ($_SESSION['USER_RIGHT_MANAGE_LABEL']==1 && isset($labelid) && $labelid != '')
[D,scripts/admin/kcfinder/core/autoload.php,72]         {
[D,scripts/admin/kcfinder/core/autoload.php,73]             $_SESSION['KCFINDER']['disabled'] = false ;
[D,scripts/admin/kcfinder/core/autoload.php,74]             $_SESSION['KCFINDER']['uploadURL'] = "{$relativeurl}/upload/labels/{$labelid}/" ;
[D,scripts/admin/kcfinder/core/autoload.php,75]             $_SESSION['KCFINDER']['uploadDir'] = "{$uploaddir}/labels/{$labelid}" ;
[D,scripts/admin/kcfinder/core/autoload.php,76]         }
[D,scripts/admin/kcfinder/core/autoload.php,77]     }
[D,scripts/admin/kcfinder/core/autoload.php,78] 
[D,scripts/admin/kcfinder/core/autoload.php,79] }
[D,scripts/admin/kcfinder/core/autoload.php,80] 
[D,scripts/admin/kcfinder/core/autoload.php,81] function kcfinder_autoload($class) {
[D,scripts/admin/kcfinder/core/autoload.php,82]     if ($class == "uploader")
[D,scripts/admin/kcfinder/core/autoload.php,83]         require "core/uploader.php";
[D,scripts/admin/kcfinder/core/autoload.php,84]     elseif ($class == "browser")
[D,scripts/admin/kcfinder/core/autoload.php,85]         require "core/browser.php";
[D,scripts/admin/kcfinder/core/autoload.php,86]     elseif (file_exists("core/types/$class.php"))
[D,scripts/admin/kcfinder/core/autoload.php,87]         require "core/types/$class.php";
[D,scripts/admin/kcfinder/core/autoload.php,88]     elseif (file_exists("lib/class_$class.php"))
[D,scripts/admin/kcfinder/core/autoload.php,89]         require "lib/class_$class.php";
[D,scripts/admin/kcfinder/core/autoload.php,90]     elseif (file_exists("lib/helper_$class.php"))
[D,scripts/admin/kcfinder/core/autoload.php,91]         require "lib/helper_$class.php";
[D,scripts/admin/kcfinder/core/autoload.php,92] }
[D,scripts/admin/kcfinder/core/autoload.php,93] 
[D,scripts/admin/kcfinder/core/autoload.php,94]     spl_autoload_register('kcfinder_autoload');
[A,scripts/admin/kcfinder/core/autoload.php,13] */
@@ -743,14 +743,14 @@ protected function output($data=null, $template=null) {
[C,scripts/admin/kcfinder/core/browser.php,743]         if ($template === null)
[C,scripts/admin/kcfinder/core/browser.php,744]             $template = $this->action;
[C,scripts/admin/kcfinder/core/browser.php,745] 
[D,scripts/admin/kcfinder/core/browser.php,746]         if (file_exists("tpl/tpl_$template.php")) {
[A,scripts/admin/kcfinder/core/browser.php,746]         if (file_exists(ROOT . "/scripts/admin/kcfinder/tpl/tpl_$template.php")) {
[C,scripts/admin/kcfinder/core/browser.php,747]             ob_start();
[C,scripts/admin/kcfinder/core/browser.php,748]             $eval = "unset(\$data);unset(\$template);unset(\$eval);";
[C,scripts/admin/kcfinder/core/browser.php,749]             $_ = $data;
[C,scripts/admin/kcfinder/core/browser.php,750]             foreach (array_keys($data) as $key)
[C,scripts/admin/kcfinder/core/browser.php,751]                 if (preg_match('/^[a-z\d_]+$/i', $key))
[C,scripts/admin/kcfinder/core/browser.php,752]                     $eval .= "\$$key=\$_['$key'];";
[D,scripts/admin/kcfinder/core/browser.php,753]             $eval .= "unset(\$_);require \"tpl/tpl_$template.php\";";
[A,scripts/admin/kcfinder/core/browser.php,753]             $eval .= "unset(\$_);require ROOT . \"/scripts/admin/kcfinder/tpl/tpl_$template.php\";";
[C,scripts/admin/kcfinder/core/browser.php,754]             eval($eval);
[C,scripts/admin/kcfinder/core/browser.php,755]             return ob_get_clean();
[C,scripts/admin/kcfinder/core/browser.php,756]         }
@@ -54,7 +54,7 @@ public function __construct() {
[C,scripts/admin/kcfinder/core/uploader.php,54]             $this->file = &$_FILES[key($_FILES)];
[C,scripts/admin/kcfinder/core/uploader.php,55] 
[C,scripts/admin/kcfinder/core/uploader.php,56]         // LOAD DEFAULT CONFIGURATION
[D,scripts/admin/kcfinder/core/uploader.php,57]         require "config.php";
